questions/10-medium-tuple-to-union/template.ts

```diff
@@ -1 +1 @@
-export type TupleToUnion<T> = any
+export type TupleToUnion<T extends any[]> = T[any]
```

Comment:
Even though TS is not complaining using any as index is less idiomatic as use number:

```suggestion
export type TupleToUnion<T extends any[]> = T[number]
```

Reply:
I agree, that definitely makes the type more specific and specific